### PR TITLE
frontend: add attestation check on device setting.

### PR DESCRIPTION
**Pull request by Beerosagos**
With this, the user can verify at any moment the authenticity of his device by checking the device view.

### CODE REVIEW

File: frontends/web/src/components/settingsButton/settingsButton.module.css [hunk 1]
```diff
@@ -17,7 +17,7 @@
     cursor: pointer;
 }
 
-.container svg {
+.container svg,img {
```
[the remark this answers is not in the document]
Comment (Beerosagos) [reply frontends/web/src/components/settingsButton/settingsButton.module.css]: Oops, you're right! Done :)

File: frontends/web/src/components/settingsButton/settingsItem.tsx [hunk 2]
```diff
@@ -3,11 +3,12 @@ import style from './settingsButton.module.css';
 
 interface SettingsItemProps {
     optionalText?: string | null;
+    optionalIcon?: (props: any) => JSX.Element;
```
[the remark this answers is not in the document]
Comment (Beerosagos) [reply frontends/web/src/components/settingsButton/settingsItem.tsx]: ok!

File: frontends/web/src/components/settingsButton/settingsButton.module.css [hunk 3]
```diff
@@ -37,7 +37,7 @@
     font-size: var(--size-small);
     color: var(--color-secondary);
     white-space: nowrap;
-    max-width: 120px;
+    max-width: 150px;
```
[the remark this answers is not in the document]
Comment (Beerosagos) [reply frontends/web/src/components/settingsButton/settingsButton.module.css:41]: AFAICT no. I tested both settings view at various screen sizes.

**Comment by benma** [comment]
seems cool - please update the CHANGELOG so we don't forget to mention it in the release notes/blog

**Comment by thisconnect** [comment]
LGTM, but requested feedback for wording. Also please add a changelog entry.

**Comment by Beerosagos** [comment]
> LGTM, but requested feedback for wording. Also please add a changelog entry.

ok :+1: changelog updated

File: frontends/web/src/locales/en/app.json [hunk 4]
```diff
@@ -404,6 +404,11 @@
       }
     },
     "hardware": {
+      "attestation": {
+        "false": "Authenticity check failed",
+        "label": "Authenticity check",
+        "true": "Your BitBox02 is authentic"
+      },
```
[the remark this answers is not in the document]
Comment (jadzeidan) [reply frontends/web/src/locales/en/app.json:411]: How about this checkmark icon?

checkcirclefilled

example:

image

[the remark this answers is not in the document]
Comment (thisconnect) [reply frontends/web/src/locales/en/app.json:411]: @jadzeidan nice, but I think the white check tick might he a tiny bit too thin, what do you think about increasing the stroke a little so it is a bit better to see?

anyway. lets merge this PR and change the icon in a different pull request. please ping me directly about the icon

**Pull request merged**